Resources/Maps/_Sunrise/Shuttles/DSO/ERT/ert_medium_gamma.yml

```
@@ -0,0 +1,2995 @@
+      parent: 1
+- proto: CentcomComputerComms
+  entities:
+  - uid: 191
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: -0.5,8.5
+      parent: 1
+- proto: CentcomTechFab
+  entities:
+  - uid: 59
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)